src/applications/disability-benefits/all-claims/tests/content/ptsdClassification.unit.spec.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import {
+  getPtsdClassification,
+  PtsdNameTitle,
+} from '../../content/ptsdClassification';
+
+describe('781 getPtsdClassification', () => {
+  it('Combat incident text is correct', () => {
```

Comment:
For all these tests, what do you think of testing both `incidentText` and `incidentTitle` since the function returns an object with both those properties?

Reply:
Makes sense, will update.